lstchain/tools/lstchain_create_dl3_file.py

```diff
@@ -45,6 +48,45 @@ class DataReductionFITSWriter(Tool):
         file_ok=True,
     ).tag(config=True)
 
+    event_filters = traits.Dict(
+        help="Enter the event filters for standard parameters - "
+        "intensity, leakage_intensity_width_2, r, wl",
+        default_value=dict(
+            {
+                "intensity": [100, np.inf],
+                "r": [0, 1],
+                "wl": [0.1, 1],
+                "leakage_intensity_width_2": [0, 0.2],
+            }
+        ),
+    ).tag(config=True)
+
+    fixed_cuts = traits.Dict(
+        help="Enter the fixed selection cut values for "
+        "gh_score(gammaness), theta and source_fov_offset",
+        default_value=dict(
+            {
+                "gh_score": 0.6,
```

Comment:
Does this dict always need to have these three values? Then you should create entries for each of them: `gammaness_cut`  `theta_cut` and `source_fov_offset`. Why is `source_fov_offset` 2.83?

Reply:
No, this is just a temporary traitlet, before we include the energy-based selection cuts. I will separate them.
The `source_fov_offset` is for applying a cut of 8 deg² to the events.